pages/en/lb4/Testing-Your-application.md

```diff
@@ -8,39 +8,236 @@ permalink: /doc/en/lb4/Testing-your-application.html
 summary:
 ---
 
-## Setup
+## Overview
 
-### With LoopBack CLI
+An automated test suite is important to ensure your application works as expected, prevent regressions as new features are added and give you confidence to refactor your codebase to keep it easy to further modify.
+
+Thorough automated testing:
+- Prevents regressions from new features/bug fixes (code introductions and refactorings) (after deployment)
+- Helps new and existing developers understand different parts of the codebase (knowledge sharing)
+- Speeds up development over the long run (the code writes itself!)
+- And much more. See [benefits of software testing](http://lmgtfy.com/?q=benefits+of+software+testing)
```

Comment:
Shouldn't be a bullet point if it's a continuation of the thought and not the list.

Reply:
Also, as much as I find it hilarious to use LMGTFY, I don't think we should be passive-aggressive about helping people see the value of test-driven development. :P

Reply:
Re
> Capitalize almost all words in a title except for conjunctions and prepositions...

We use sentence case for headings; cf. http://loopback.io/doc/en/contrib/Doc-style-guide.html#headings